ui/components/app/gas-customization/gas-modal-page-container/gas-modal-page-container.component.js

```diff
@@ -57,13 +57,18 @@ export default class GasModalPageContainer extends Component {
         disconnectGasFeeEstimatePoller(pollingToken);
       }
     });
+    window.addEventListener('beforeunload', this._beforeUnload);
   }
 
-  componentWillUnmount() {
+  _beforeUnload = () => {
     this._isMounted = false;
```

Comment:
It seems like this should still be set upon unmount.

Reply:
yes, good catch